Consistency improvements, unification of comments & typo fixes (#104)
grammyjs/website: site/docs/guide/filter-queries.md

```ts
bot.on("message", (ctx) => {

>   // Could be undefined if the received message has no text.

Author: I think @KnorpelSenf suggested to not say `text` or `Context` to be more human-friendly

>   // Could be undefined if the received message has no text.

Author: I think @KnorpelSenf suggested to not say text or Context to be more human-friendly.

  const text: string | undefined = ctx.msg.text;
});
bot.on("message:text", (ctx) => {

>   // Text is always defined because this handler is called when a text message is received.

Author: Included? Doesn't that change the meaning?

>   // Text is always defined because this handler is called when a text message is received.

Reviewer: Yes. If you don't want to use _defined_, we could say _present_, or _required_, but _included_ is not the right terminology to differentiate between the types `text?: string` and `text: string`.

Simple filters for updates, and sub-filters:

```ts

> bot.on("message"); // called for any type of message

Reviewer 2: ```suggestion
bot.on("message"); // called when any message is received
```

grammY will then search through different values to match your query.

```ts

> bot.on(":text"); // any text messages and any text post of channels

Reviewer 2: ```suggestion
bot.on(":text"); // any text messages and any channel posts with text
```
or:
```suggestion
bot.on(":text"); // any text messages and any textual channel posts
```

Author: Posts that aren't text posts, can't include text. For example, the photo posts can't have text, but they'll have captions. I think saying text posts is better.

Reviewer: What was even wrong with text channel posts? We also have text messages …

In other words, using `msg` is equivalent to listening for both `'message'` and `'channel_post'` events.

```ts

> bot.on("msg"); // any message or channel post

Reviewer 2: ```suggestion
bot.on("msg"); // any messages or channel posts
```

This `edit` shortcut groups edited messages and edited channel posts.

```ts

> bot.on("edit"); // any message or channel post edit

Reviewer 2: ### 
```suggestion
bot.on("edit"); // any edits of messages or channel posts
```

Author: Isn't "edit of messages" better?

The `:media` shortcut groups photo and video messages.

```ts
bot.on("message:media"); // photo and video messages
bot.on("edited_channel_post:media"); // edited channel posts with media

> bot.on(":media"); // new media message or channel posts

Reviewer 2: ```suggestion
bot.on(":media"); // media messages or channel posts
```

```ts
bot.on("message:new_chat_members:is_bot"); // a bot joined the chat

> bot.on("message:left_chat_member:me"); // your bot left a chat (or was removed)

Reviewer: This changes the meaning. Are you sure the updates are received when a bot leaves by itself? I think you only get those messages when someone removes your bot, but I'm not sure.
